src/openvic-simulation/Modifier.cpp

```diff
@@ -191,12 +193,13 @@ bool ModifierManager::setup_modifier_effects() {
 	ret &= add_modifier_effect("ruling_party_support", true);
 	ret &= add_modifier_effect("social_reform_desire", false);
 	ret &= add_modifier_effect("supply_consumption", false);
+	ret &= add_modifier_effect("tax_efficiency", true);
 	ret &= add_modifier_effect("army_tech_research_bonus", true);
 	ret &= add_modifier_effect("commerce_tech_research_bonus", true);
 	ret &= add_modifier_effect("culture_tech_research_bonus", true);
 	ret &= add_modifier_effect("industry_tech_research_bonus", true);
-	ret &= add_modifier_effect("naval_tech_research_bonus", true);
-	ret &= add_modifier_effect("unit_start_experience", true); // weird, naval_unit_start_experience = 15 would give a 15% boost
+	ret &= add_modifier_effect("navy_tech_research_bonus", true);
```

Comment:
If you don't want to hard-code the building max modifiers, you may not want to hard-code these either as they are derived from the names of the tech categories. Does any mod add a tech category? That I don't know.

Reply:
Thanks for letting me know, dunno about mods changing categories but since we're making it possible to change them then the changes should propagate all the way through.